services/121-service/src/scoped.repository.ts

```diff
@@ -49,6 +45,12 @@ type EntityRelations = Record<string, string[]>;
 // Define here any entities that do have an INDIRECT relation to registration
 const indirectRelationConfig: EntityRelations = {
   IntersolveVisaWalletEntity: ['intersolveVisaCustomer', 'registration'],
+  IntersolveVisaChildWalletEntity: ['intersolveVisaCustomer', 'registration'],
```

Comment:
Should this one also have intersolveVisaParentWallet?

Reply:
Good catch switched it around